contracts/utils/ChainTracker.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IChainTracker } from '../interfaces/IChainTracker.sol';
+
+abstract contract ChainTracker is IChainTracker {
+    // uint256(keccak256('chain_tracker_slot')) - 1
+    bytes32 internal constant CHAIN_TRACKER_SLOT = 0x6c51f1695661c243e1b0c3337f0f4c4f45116119470fb2cb0eb05fd857f16ac2;
```

Comment:
create a `TestChainTracker` contract to test the hash derivation like we do for other contracts?